app/serializers/task_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class TaskSerializer < ActiveModel::Serializer
+  attributes :id, :name, :description, :state, :expired_at
+  belongs_to :author
+  belongs_to :assignee
+
+  def id
+    object.id.to_s
```

Comment:
any reason to have string id?

Reply:
it works without stringify. may be it is useful when we want to be sure the result is always a string?